bazel/repository_locations.bzl

```diff
@@ -598,25 +598,25 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         project_name = "Protocol Buffers",
         project_desc = "Language-neutral, platform-neutral extensible mechanism for serializing structured data",
         project_url = "https://developers.google.com/protocol-buffers",
-        version = "3.14.0",
-        sha256 = "6dd0f6b20094910fbb7f1f7908688df01af2d4f6c5c21331b9f636048674aebf",
+        version = "3.16.0",
+        sha256 = "d7371dc2d46fddac1af8cb27c0394554b068768fc79ecaf5be1a1863e8ff3392",
         strip_prefix = "protobuf-{version}",
         urls = ["https://github.com/protocolbuffers/protobuf/releases/download/v{version}/protobuf-all-{version}.tar.gz"],
         use_category = ["dataplane_core", "controlplane"],
-        release_date = "2020-11-13",
+        release_date = "2021-05-06",
```

Comment:
3.17.0 was released 2 days ago - https://github.com/protocolbuffers/protobuf/releases/tag/v3.17.0. Any interest in jumping to that version or do in a follow up?

Reply:
Sure, I can bump it now.

Reply:
And it turns out 3.17.0 has a regression, so that won't work. https://github.com/protocolbuffers/protobuf/issues/8611